src/controllers/species.controller.ts

```diff
@@ -86,6 +88,25 @@ export class SpeciesController {
     await this.speciesRepository.updateById(id, species);
   }
 
+  @post('/species/combine', {
+    responses: {
+      '204': {
+        description: 'Species POST success',
+      },
+    },
+  })
+  async combine(
+    @requestBody() request: { combine: number[]; species: Species },
+  ): Promise<void> {
+    const newSpecies = await this.speciesRepository.create(request.species);
+    console.log(newSpecies);
+    for (const id of request.combine) {
+      const query = `UPDATE trees SET species_id=${newSpecies.id} WHERE species_id=${id}`;
+      await this.treesRepository.execute(query, []);
+      await this.speciesRepository.deleteById(id);
```

Comment:
Unfortunately, this won't work in test and production environments, due to deletes being prohibited (issue #486). We should really set `active=false`.

Reply:
Makes sense. Although active is an integer column. Are we using 0 for inactive and 1 for active?

Reply:
I think that must have been implemented incorrectly - it should be a Boolean. We'll need to raise a bug against the database project (treetracker-database) and create a database migration script to correct it.

Reply:
I talked to @deepwinter and filed an issue there. I'm sending another commit with the updated code but we should wait until changes are made to the database to merge it.